server/src/main/java/org/elasticsearch/action/admin/indices/rollover/RolloverConditions.java

```diff
@@ -256,6 +258,60 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return builder;
     }
 
+    /**
+     * Parses a cluster setting configuration, it expects it to have the following format: "condition1=value1,condition2=value2"
+     * @throws ElasticsearchParseException if the input is invalid, if there are unknown conditions or invalid format values.
+     */
+    public static RolloverConditions parseSetting(String input, String setting) {
+        String[] sConditions = input.split(",");
```

Comment:
Should we validate against empty (`""`) `input` here?

Reply:
I will add a specific check, right now it's caught by the next split call.

Reply:
Added